### PR TITLE
docs: fix toMatchSnapshot optional name usage

### LINKED ISSUE #13319: [docs]: add toMatchSnapshot without a name
https://playwright.dev/docs/test-assertions#screenshot-assertions-to-match-snapshot

### PR DESCRIPTION
**Pull request by mxschmitt**
Fixes https://github.com/microsoft/playwright/issues/13319

### CODE REVIEW

File: docs/src/release-notes-js.md [hunk 1]
```diff
@@ -22,7 +22,6 @@ title: "Release notes"
 
   ```js
   expect(await page.screenshot()).toMatchSnapshot({
-    fullPage: true, // take a full page screenshot
```
Comment (mxschmitt) [inline docs/src/release-notes-js.md:25]: drive-by: since this can't work.

**Review by aslushnikov — APPROVED** [review]
